packages/contracts/contracts/ChugSplashManager.sol

```diff
@@ -364,6 +376,37 @@ contract ChugSplashManager is Owned {
         emit ProxySetToTarget(_name, _proxy, _proxyType, _name);
     }
 
+    /**
+     * @notice Allows the project owner to set an Executor Selection Strategy, which determines the
+     *         strategy used to select executors for this project. Only callable when there is no
+     *         active bundle.
+     */
+    function setExecutorSelectionStategy(IExecutorSelectionStrategy _executorSelectionStrategy)
+        external
+        onlyOwner
+    {
+        require(
+            activeBundleId == bytes32(0),
+            "ChugSplashManager: a bundle has been approved and not yet completed"
+        );
+        // TODO: Add an EIP-165 check for the ESS address here?
+        executorSelectionStrategy = _executorSelectionStrategy;
+    }
+
+    /**
+     * @notice Get the address of the executor selected by the Executor Selection Strategy to
+     *         execute a given bundle ID.
+     *
+     * @param _bundleId Bundle ID.
+     */
+    function getSelectedExecutor(bytes32 _bundleId) public view returns (address) {
+        address executor = executorSelectionStrategy.getSelectedExecutor(
```

Comment:
Minor simplification, can just return the result of the function directly at this point